### DISCUSSION TITLE
KeyError 982

### DISCUSSION THREAD

**Discussion by dtxn** [vladmandic/sdnext#648] [Q&A] [answered]
Hi. After I sent image from Text to Inpaint page, in Inpaint page positive and negative prompt windows appeared in "forever loading" status. I restarted user interface from Settings menu and then after that keyboard and mouse stopped reacting on any presses in Vlad app.  And that error with keyboard happened not the first time. Earlier it was even when pressing randomly selected keyboard key, every time it showed different letter :) 
![image](https://user-images.githubusercontent.com/132169795/235362036-514f5548-6431-4551-90e6-c67dd35be1f1.png)

**Comment by vladmandic** [comment, marked as answer]
see pinned item on top of discussions: #224
